components/automate-ui/src/app/pages/team-details/team-details.component.html

```diff
@@ -9,86 +9,72 @@
+      </table>
+
+      <chef-tab-selector (change)="onSelectedTab($event)" [value]="tabValue">
+        <chef-option value='users' routerLink="./" fragment="users">Users</chef-option>
+        <chef-option value='details' routerLink="./" fragment="details">Details</chef-option>
       </chef-tab-selector>
     </chef-page-header>
-    <section class="page-body" *ngIf="team?.id">
-      <div id="users-list">
-        <div *ngIf="!teamMembershipView">
-          <!-- TODO remove [overridePermissionsCheck]=true once the UI is able to check specific objects
-          since the path to permission on here is /auth/teams/:id/users we'll have to show everything for now. -->
-          <app-user-table
-            [addButtonText]="addButtonText"
-            [removeText]="removeText"
-            [users$]="sortedUsers$"
-            [addButtonEnabled]="!editMode"
-            [baseUrl]="'/auth/users/' + team.id"
-            [overridePermissionsCheck]=true
-            (addClicked)="toggleUserMembershipView()"
-            (removeClicked)="removeUser($event)">
-          </app-user-table>
-        </div>
-        <div *ngIf="teamMembershipView">
-          <app-user-team-membership-table
-            [usersToFilter]="sortedUsers$"
-            (addClicked)="addUsers($event)"
-            (cancelClicked)="toggleUserMembershipView()">
-          </app-user-team-membership-table>
+    <ng-container *ngIf="tabValue === 'users'">
+      <section class="page-body" *ngIf="team?.id">
+        <div id="users-list">
+          <div *ngIf="!teamMembershipView">
+            <!-- TODO remove [overridePermissionsCheck]=true once the UI is able to check specific objects
+            since the path to permission on here is /auth/teams/:id/users we'll have to show everything for now. -->
+            <app-user-table
+              [addButtonText]="addButtonText"
+              [removeText]="removeText"
+              [users$]="sortedUsers$"
+              [addButtonEnabled]=true
+              [baseUrl]="'/auth/users/' + team.id"
+              [overridePermissionsCheck]=true
+              (addClicked)="toggleUserMembershipView()"
+              (removeClicked)="removeUser($event)">
+            </app-user-table>
+          </div>
+          <div *ngIf="teamMembershipView">
+            <app-user-team-membership-table
+              [usersToFilter]="sortedUsers$"
+              (addClicked)="addUsers($event)"
+              (cancelClicked)="toggleUserMembershipView()">
+            </app-user-team-membership-table>
+          </div>
         </div>
-      </div>
-    </section>
+      </section>
+    </ng-container>
+    <ng-container *ngIf="tabValue === 'details'">
+      <section class="page-body">
+        <form [formGroup]="updateNameForm">
+          <chef-form-field>
+            <label *ngIf="isV1">Description *</label>
+            <label *ngIf="!isV1">Name *</label>
+            <input chefInput formControlName="name" type="text" (keyup)="keyPressed()">
```

Comment:
Needs an associated error element:
```
<chef-error *ngIf="(updateNameForm.get('name').hasError('required') || updateNameForm.get('name').hasError('pattern')) && updateNameForm.get('name').dirty">
  {{ descriptionOrName() }} is required.
</chef-error>
```
Note that because the way @susanev 	wants these error messages to be (`<displayed-name> is required`), you might want to have a property `descriptionOrName()` as `return isV1? 'Description' : 'Name';`. With that in place, then L65-66 above can be simplified to a single line:
```
<label>{{ descriptionOrName() }} <span aria-hidden="true">*</span></label>
```